File: Reducebutgrow.js
const grow = (x) => x.reduce((a, b) => a * b);
// console.log(grow([1, 2, 3, 4, 5]), 120);
/*
  Reducebutgrow
*/
console.log(grow([1, 2, 3]), 6);
console.log(grow([4, 1, 1, 1, 4]), 16);
console.log(grow([2, 2, 2, 2, 2, 2]), 64);
console.log(grow([1, 2, 3, 4, 5]), 120);

File: reducebutgrow.py
import math
def grow(arr):
    return math.prod(arr)

from functools import reduce

def grow(arr):
     return reduce(lambda x, y: x * y, arr)

  
# def grow(arr):
#    product = 1
#    for i in arr():
#        product *= i
#    return product


print(grow( [1, 2, 3]), 6),
print(grow( [1, 2, 3, 4]), 24),
print(grow([2,2,2,2,2,2,2,2,2,2,2]),2048),   

